doc/03-Configuration.md

```diff
@@ -0,0 +1,28 @@
+# Configuration
+
+This is how you configure your Icinga DB so it works right off the bat.
```

Comment:
We're configuring the `Icinga DB web module` here. Also, it should mention that it comes with default settings.

Reply:
makes sense!